modules/install-sno-installing-with-usb-media.adoc

```diff
@@ -0,0 +1,52 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="installing-with-usb-media_{context}"]
+= Installing with USB media
+
+Installing with USB media involves creating a bootable USB drive with the discovery ISO on the administration node. Booting the server with the USB drive prepares the node to install singe node OpenShift with the Assisted Installer (AI).
```

Comment:
no "single node OpenShift" anywhere.  If you want a  suggested replacement, please let me know.